nx-X11/programs/Xserver/include/scrnintstr.h

```diff
@@ -549,6 +549,7 @@ typedef struct _Screen {
     void *		devPrivate;
     short       	numVisuals;
     VisualPtr		visuals;
+    WindowPtr		root;
     int			WindowPrivateLen;
```

Comment:
Formatting in scrnintstr.h (WindowPtr root;) is not really broken. The breakage is caused due to Git displaying tabs in a non-consistent way in diffs.